### PR TITLE
Heliostation sm waste and scrubber pipe layer fixes

### PR DESCRIPTION
**Pull request by EIB0N**
## About The Pull Request

This changed the smart pipes for the scrubbers and exhaust from layer 3 to layer 2 like the rest of the scrubbers around the station

## Why It's Good For The Game

Help out the Engineering boys be more organized

## Changelog
:cl:
fix: SM Scrubber and exhaust are now layer 2!
/:cl:

### CODE REVIEW

**Comment by Helianthus00** [comment]
image
As far as I know, this does not work. A far better solution is to just change the color to something else that is not used in this area. Then I'd keep the rest of the pipes on layer 3 as normal so that the rest of the layers are reserved for extra pipes.

**Review by Helianthus00 — CHANGES_REQUESTED** [review]
image
As far as I know, this doesn't work. You need to change the color of the pipes in the wastenet instead and leave everything else on layer 3.

**Comment by EIB0N** [comment]
Fixed it

**Review by Helianthus00 — CHANGES_REQUESTED** [review]
image
This gas meter is on the wrong layer. Please make sure you test this as well to make sure there aren't unintended issues. I still don't like the use of layer 2 scrubber pipes in the chamber.

**Review by EIB0N — COMMENTED** [review]
gas flow meter changed to layer 2

**Review by Helianthus00 — CHANGES_REQUESTED** [review]
image
Making this pipe brown means it will no longer connect with the red pipes. You need to change it back to red or it won't work.

File: tgstation.dme [hunk 1]
```diff
@@ -3637,7 +3637,6 @@
 #include "fulp_modules\maps\cytology_spawner\beefcyto.dm"
 #include "fulp_modules\maps\fulp_shuttles\fulp_shuttles.dm"
 #include "fulp_modules\ZFulpcode\modules\jobs\huds.dm"
-#include "fulp_modules\ZFulpcode\modules\jobs\inventories.dm"
```
Comment (Helianthus00) [inline tgstation.dme:3640]: This shouldn't be removed.

**Review by Helianthus00 — CHANGES_REQUESTED** [review]
You don't need pipe bridges here. They are only needed if the pipes are the same layer.

File: tgstation.dme (hunk 1 above)
Comment (EIB0N) [reply tgstation.dme:3640]: readded it

**Review by EIB0N — COMMENTED** [review]
Fixed the pipe colors and removed the bridges

File: tgstation.dme (hunk 1 above)
Comment (Helianthus00) [reply tgstation.dme:3640]: it needs to be in alphabetical order

**Review by Helianthus00 — APPROVED** [review]

**Pull request merged**